src/core/link-to-dfn.js

```diff
@@ -92,12 +92,13 @@ export async function run(conf, doc, cb) {
           // data-lt[0] serves as unique id for the dfn which this element references
           ant.dataset.xref = lt[0];
         } else {
-          const frag = "#" + encodeURIComponent($dfn.prop("id"));
-          $ant.attr("href", frag).addClass("internalDFN");
+          const frag = "#" + encodeURIComponent(dfn.id);
```

Comment:
I think we should just do:

ant.href = "#" + encodeURIComponent(dfn.id);

Reply:
update the above.

Reply:
no need to `encodeURIComponent`?

Reply:
when we set it via the .href, it automatically happens.